bench/wt2853_perf/main.c

```diff
@@ -262,7 +262,12 @@ thread_insert(void *arg)
             flag = 0;
         }
         maincur->set_value(maincur, post, bal, extra, flag, key);
-        testutil_check(maincur->insert(maincur));
+        ret = maincur->insert(maincur);
+        if (ret == WT_ROLLBACK) {
+            testutil_check(session->rollback_transaction(session, NULL));
+            continue;
+        }
+        testutil_assert(ret == 0 || ret == WT_ROLLBACK);
```

Comment:
You can just assert that `ret == 0`.  `ret` cannot be `WT_ROLLBACK` at this point.